test/e2e/framework/framework.go

```diff
@@ -107,6 +112,11 @@ func (f *Framework) setup() error {
 			return fmt.Errorf("fail to setup aws: %v", err)
 		}
 	}
+	if err := f.SetupEtcdBackupOperator(); err != nil {
```

Comment:
```
err = Delete()
if err != nil {...
```

Reply:
hmm, isn't `if err := f.SetupEtcdBackupOperator(); err != nil {` shorter since it is one liner.

Reply:
It's the current convention.
We should discourage any code like that:
```
if xxx := xxx(); xxx {
}
```

Reply:
maybe I am missing something.
all setup functions has the following style.
```go
if xxx := xxx(); xxx {
}
```

```go
func (f *Framework) setup() error {
	if err := f.setupStorageClass(); err != nil {
		return fmt.Errorf("failed to setup storageclass(%v): %v", f.StorageClassName, err)
	}
	if err := f.SetupEtcdOperator(); err != nil {
		return fmt.Errorf("failed to setup etcd operator: %v", err)
	}
	logrus.Info("etcd operator created successfully")
	if os.Getenv("AWS_TEST_ENABLED") == "true" {
		if err := f.setupAWS(); err != nil {
			return fmt.Errorf("fail to setup aws: %v", err)
		}
	}
	if err := f.SetupEtcdBackupOperator(); err != nil {
		return fmt.Errorf("failed to create etcd backup operator: %v", err)
	}
	logrus.Info("etcd backup operator created successfully")

	logrus.Info("e2e setup successfully")
	return nil
}
```

Anyways, I think keeping the code consistent is the key here.

Reply:
That should be discouraged

Reply:
alright then.